src/vma/dev/ib_ctx_handler_collection.h

```diff
@@ -46,16 +46,18 @@ class ib_ctx_handler_collection
 public:
 	ib_ctx_handler_collection();
 	~ib_ctx_handler_collection();
-	void            map_ib_devices(); //return num_devices, can use rdma_get_devices()
+
 	ib_ctx_handler* get_ib_ctx(struct ibv_context*);
 	ib_ctx_handler* get_ib_ctx(const char *ifa_name);
-	size_t		get_num_devices() {return m_n_num_devices; };
-	size_t          mem_reg_on_all_devices(void* addr, size_t length,
-			ibv_mr** mr_array, size_t mr_array_sz,
-			uint64_t access);
+	size_t get_num_devices() {return m_n_num_devices; };
+	size_t mem_reg_on_all_devices(void* addr, size_t length,
+			ibv_mr** mr_array, size_t mr_array_sz, uint64_t access);
 	ts_conversion_mode_t    get_ctx_time_conversion_mode();
 
 private:
+	void update_tbl();
+	void print_val_tbl();
+
```

Comment:
I dont like this renaming - The api is for classes which inherint from netlink_socket_mgr but this is not the case here.